backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientDataResolver.java

```diff
@@ -66,6 +80,12 @@ public CompletableFuture<List<PhoneNumber>> getPhoneNumbers(
     return patientPhoneNumbersLoader.load(person.getInternalId());
   }
 
+  @SchemaMapping(typeName = "Patient", field = "telephone")
+  public CompletableFuture<PhoneNumber> getPrimaryPhoneNumbers(
+      Person person, DataLoader<UUID, PhoneNumber> patientPrimaryPhoneNumberLoader) {
+    return patientPrimaryPhoneNumberLoader.load(person.getInternalId());
+  }
```

Comment:
Adding this back since it should function correctly now and it will improve performance